docs/src/components/InteractiveExample/index.tsx

```diff
@@ -32,7 +100,8 @@ export default function InteractiveExample({
 }: Props) {
   const [_, copy] = useCopyToClipboard();
   const [key, setKey] = React.useState(0);
-  const [showPreview, setShowPreview] = React.useState(!showCode);
+  const [tab, setTab] = React.useState<Tab>(Tab.PREVIEW);
+  const [jsxCode, setJsxCode] = React.useState(compileTSXtoJSX(src));
```

Comment:
To avoid calling the `compileTSXtoJSX` on every render you can use this syntax
```suggestion
  const [jsxCode, setJsxCode] = React.useState(() => compileTSXtoJSX(src));
```